Cargo.toml

```diff
@@ -29,6 +29,7 @@ num-rational = "0.4.2"
 default = ["flac", "vorbis", "wav", "mp3", "cpal"]
```

Comment:
we got a ton of features, I was wondering about renaming `cpal` to `playback-cpal`. That makes clear what its for and if we add another playback backend in the future we can name that `playback-other`.

I would also be okay with plane `playback` I do not think the end user needs to know what we are using.

Reply:
@dvdsk renamed to `playback`. I'd rather add feature descriptions to `README.md`.